proof-market/application-developers/tutorials/signing-up.mdx

```diff
@@ -0,0 +1,39 @@
+import ApiSchema from '@theme/ApiSchema';
+
+# Signing up
+
+This tutorial showcases how authorization works in Proof Market and its API.
+
+:::info[Metamask]
+
+Authorization inside Proof Market requires a Metamask wallet.
```

Comment:
Well it's not metamask wallet, it's any eth compatible wallet using the private key to sign